src/ipinfo.erl

```diff
@@ -74,9 +77,17 @@ create(AccessToken) ->
 create(AccessToken, Settings) ->
     CountriesFile = get_config(countries, Settings,
         filename:join(code:priv_dir(ipinfo), ?DEFAULT_COUNTRY_FILE)),
+    EuCountriesFile = get_config(eu_countries, Settings,
+        filename:join(code:priv_dir(ipinfo), ?DEFAULT_EU_COUNTRY_FILE)),
     BaseUrl = get_config(base_url, Settings, ?DEFAULT_BASE_URL),
     Timeout = get_config(timeout, Settings, ?DEFAULT_TIMEOUT),
     CacheTtl = get_config(cache_ttl, Settings, ?DEFAULT_CACHE_TTL_SECONDS),
+    EuCountries = case prepare_countries(EuCountriesFile) of 
+        {ok, EuCountriesOutput} ->
+            EuCountriesOutput;
+        {error, ErrorReason} ->
+            {error, ErrorReason}
+    end,
```

Comment:
It's the wrong way to do it - this will just assign `{error, ErrorReason}` to `EuCountries` in case of error, but continue as usual.